onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -294,7 +295,7 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
       transformers.emplace_back(std::make_unique<MatMulIntegerToFloatFusion>(cpu_ep));
       transformers.emplace_back(std::make_unique<DynamicQuantizeMatMulFusion>(cpu_ep));
 
-      transformers.emplace_back(std::make_unique<ConvActivationFusion>(cpu_cuda_rocm_acl_armnn_eps));
+      transformers.emplace_back(std::make_unique<ConvActivationFusion>(cpu_cuda_rocm_acl_armnn_js_eps));
```

Comment:
Does this do the fusion with ConvTranspose or is another optimizer and new unit tests required?

Reply:
Currently ConvActivationFusion only works on Conv operator. For fusing ConvTranspose nodes with activation we need to add a new transformation, ConvTransposeActivationFusion (not included in this PR). The JSEP code works if the input graph has already has ConvTranspose and actions fused.

Reply:
Currently this is only for Conv. For ConvTranspose I just need to register selection-actions pair using the same Selector and Actor used for Conv only using opcode "CovTranspose" instead of "Conv".

Reply:
Adding ConvTranspose also.

Reply:
Backed out ConvTranspose

Reply:
> Backed out ConvTranspose

is it still here?

https://github.com/microsoft/onnxruntime/blob/1834ab8512128c06626e70a09a4fb441912fa4db/onnxruntime/core/optimizer/conv_activation_fusion.cc#L270